cards/cards.css

```diff
@@ -0,0 +1,160 @@
+:root{
+    --bkg-clr:linear-gradient(45deg,aqua  0%, rgba(182, 241, 252, 1) 50%, rgba(252, 174, 241, 1) 70%);
+    --sec-text-clr:gray;
+    --pri-text-clr:black;
+    --danger-clr:red;
+}
```

Comment:
similary use secondary and primary color

Reply:
changed to secondary and primary